### PR TITLE
Harmonize client config format with exporter config

### PR DESCRIPTION
**Pull request by NickCao**

### CODE REVIEW

**Comment by NickCao** [comment]
https://github.com/jumpstarter-dev/jumpstarter-controller/actions/runs/11388223376

File: examples/soc-pytest/jumpstarter_example_soc_pytest/test_on_rpi4.py [hunk 1]
```diff
@@ -52,8 +53,9 @@ def test_setup_device(self, client, console):
         try:
             client.dutlink.storage.write_local_file("image/images/latest.raw")
         except opendal.exceptions.NotFound:
-            pytest.exit("No image found, please enter the image directory and run `make`, "
-                        "more details in the README.md")
+            pytest.exit(
+                "No image found, please enter the image directory and run `make`, " "more details in the README.md"
```
Comment (mangelajo) [inline examples/soc-pytest/jumpstarter_example_soc_pytest/test_on_rpi4.py:57]:
```suggestion
                "No image found, please enter the image directory and run `make`, more details in the README.md"
```

**Review by mangelajo — APPROVED** [review]
Nice!

**Comment by mangelajo** [comment]
crap this was on top of grpcs change

**Comment by mangelajo** [comment]
Ok, I manually rebased and applied to main.

**Pull request merged**